README.md

```diff
@@ -8,12 +8,15 @@ Please read [contribution guidelines](contributing.md), thank you!
 
 ## Contents
 
-- [Programming Language](#programming-language)
-- [Web Development](#web-development)
-- [Mobile Development](#mobile-development)
-- [Game Development](#game-development)
-- [Miscellaneous](#miscellaneous)
-- [Articles](#articles)
+- [Awesome Roadmaps ![Awesome](https://awesome.re) [![Build Status](https://travis-ci.org/liuchong/awesome-roadmaps.svg?branch=master)](https://travis-ci.org/liuchong/awesome-roadmaps)](#awesome-roadmaps-img-srchttpsawesomerebadgesvg-altawesome-img-srchttpstravis-ciorgliuchongawesome-roadmapssvgbranchmaster-altbuild-status)
```

Comment:
seems something wrong here?

Reply:
yeah, it took some time for me to figure out that my VS Code extension was changing it automatically. I'm sorry. Fixed

Reply:
Thanks! ❤️